drivers/hmis/spec/requests/hmis/project_spec.rb

```diff
@@ -123,6 +133,85 @@
       expect(result.dig('data', 'project', 'access')).to include('canDeleteProject' => true)
     end
   end
+
+  describe 'project assessments query' do
+    before(:each) do
+      hmis_login(user)
+    end
+
+    let(:project_assessments_query) do
+      <<~GRAPHQL
+        query GetProjectAssessments(
+          $id: ID!
+          $limit: Int = 10
+          $offset: Int = 0
+        ) {
+          project(id: $id) {
+            assessments(
+              limit: $limit
+              offset: $offset
+            ) {
+              offset
+              limit
+              nodesCount
+              nodes {
+                id
+                inProgress
+                role
+                definition {
+                  title
+                }
+                enrollment {
+                  entryDate
+                  exitDate
+                }
+              }
+            }
+          }
+        }
+      GRAPHQL
+    end
+
+    it 'resolves both WIP and non-WIP assessments on the project' do
+      response, result = post_graphql(id: p1.id) { project_assessments_query }
+      expect(response.status).to eq 200
+      records = result.dig('data', 'project', 'assessments', 'nodes')
+      expect(records.size).to eq(4)
+      expect(records.pluck('inProgress')).to contain_exactly(true, false, true, false)
```

Comment:
This is okay, but I think it's more robust to expect the exact assessments that should be returned
```suggestion
      expect(records.pluck('id')).to contain_exactly(a1.id.to_s, a2.id.to_s, a3.id.to_s, a4.id.to_s)
```